src/book/01-team/02-joining/03-induction.mdx

```diff
@@ -6,14 +6,34 @@ description: The NUbots lab induction process for new members.
+
+## EE/ES Lab Induction
+
+After reading the safety documentation, you are required to complete the EE/ES Lab Induction. This can be accessed by clicking the link in step 2 on Canvas. Once you have finished the quiz, navigate back to the NUbots induction page - if you instead click 'next' in Canvas, it will take you to further quizzes for Electrical Engineering, which you are **not** required to complete.
+
+You must score **100%** in the quiz. The [Electrical Engineering Safety Manual](https://www-eng.newcastle.edu.au/eecs/ect/oh&s/) accompanies the quiz. **This quiz must be completed by all team members each year**, even if you already have swipe card access.
+
+Students working in the EE labs for their coursework will have already completed this quiz. If you have already completed the quiz in this calender year, you do not need to do it again.
 
 ## The NUbots Lab Induction
 
-The second part of the process is the NUbots lab induction. This is an in-person induction done with a Team Leader or Team Safety Officer in the lab. This induction walks new members though the available safety and emergency equipment in the lab, how to safely handle batteries and robots, and what to do when there is an emergency.
+The next part of the process is the NUbots lab induction. This is an in-person induction done with a Team Leader or Team Safety Officer in the lab. This induction walks new members though the available safety and emergency equipment in the lab, how to safely handle batteries and robots, and what to do when there is an emergency.
+
+Upon completion of the NUbots lab induction, new members must must fill out the [lab induction checklist](./assets/lab-induction-checklist.pdf). They are to obtain a digital copy to upload to Canvas, and the hardcopy will be kept in the lab as a record. A copy of the checklist will be provided to the student at the beginning of the induction.
+
+## Request Swipe Card Access
+
+The final step on Canvas involves an embedded app that allows you to request access. After signing in to the app with your student details, you will see a form. Select the NUbots access group and choose Alex Mendes to approve the request. If any steps are not complete, the app will indicate this. Once the steps are all complete, you can submit your request for swipe-card access.
 
-Upon completion of the NUbots lab induction, new members have to fill-out, sign, and submit the [lab induction checklist](./assets/lab-induction-checklist.pdf) to the demonstrator, as a record of their completion. A copy of this checklist will be provided during the induction.
+It may take a few days for the application to be approved and the request sent to IFS to enable swipe-card access. Once this is done, you can use your student card to access the external doors of ES and the NUbots lab between the hours of 7am and 10pm, any day of the week.
```

Comment:
I had to google IFS I think we should expand this out

```suggestion
It may take a few days for the application to be approved and the request sent to Infrastructure and Facilities Services in order to enable swipe-card access. Once this is done, you can use your student card to access the external doors of the ES building and the NUbots lab between the hours of 7am and 10pm, any day of the week.
```